Gemfile

```diff
@@ -47,6 +47,7 @@ group(:test) do
   gem "rspec", "~> 3.1", require: false
   gem "rspec-expectations", ["~> 3.9", "!= 3.9.3"]
   gem "rspec-its", "~> 1.1", require: false
+  gem 'rspec-mocks', '< 3.13.3', require: false # breaking change afterwards: https://github.com/rspec/rspec/blob/rspec-mocks-v3.13.5/rspec-mocks/Changelog.md#3133--2025-05-01
```

Comment:
maybe the same as here?
https://github.com/OpenVoxProject/puppet/pull/129#discussion_r2156736786

Reply:
done